src/js/readonly.js

```diff
@@ -12,6 +12,7 @@ export default {
         const nodes = this.form.getRelatedNodes( 'readonly', '', updated ).get();
         nodes.forEach( node =>  {
             node.closest( '.question' ).classList.add( 'readonly' );
+            node.setAttribute( 'aria-hidden', 'true' );
```

Comment:
Just repeating my earlier question. Should all readonly inputs really be aria-hidden? It's very common to show a calculated value in a readonly input field. I just want to make sure.

Reply:
I'm not certain if all read-only fields are hidden, but it seems those with *class* `empty` are hidden for sure https://github.com/enketo/enketo-core/blob/78a971943c2e1b4d7c85c618a6522c8be538941e/src/sass/core/_main.scss#L297

going to change this to check if *class* `empty` is present in the node